tests/virtualization/vagrant/add_box.pm

```diff
@@ -16,23 +16,15 @@
 # Summary: Test for vagrant and packaged addons
 # Maintainer: dancermak <dcermak@suse.com>
 
+use base "consoletest";
 use strict;
 use warnings;
-use base "basetest";
 use testapi;
 use utils;
+use vagrant;
 
 sub run() {
-    select_console('root-console');
-
-    zypper_call('in vagrant virtualbox vagrant-libvirt');
-
-    assert_script_run('systemctl start vboxdrv');
-    assert_script_run('systemctl start vboxautostart');
-    assert_script_run('systemctl start libvirtd');
-
-    assert_script_run('usermod -a -G vboxusers bernhard');
-    assert_script_run('usermod -a -G libvirt bernhard');
+    setup_vagrant(1);
```

Comment:
Probably better to have a named parameter instead of `1`. Something like `install_virtualbox => true`

Reply:
Yes, I'll change that.

Reply:
There is another occurrence in the other file.